src/mainboard/protectli/vault_adl_n/variants/vp32xx/overridetree.cb

```diff
@@ -0,0 +1,69 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	device domain 0 on
+		subsystemid 0x8086 0x7270 inherit
+
+		device ref igpu on
+			subsystemid 0x8086 0x2212
+			register "ddi_portA_config" = "1" # HDMI on port A
+			register "ddi_portB_config" = "1" # DP on port B
+			register "ddi_ports_config" = "{
+				[DDI_PORT_A] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
```

Comment:
Now that I look at it, the DDC is not used on Port A on this platform per newest schematics, but it is used on older shcematics. I will update it on the rc2 fixes branch for VP32xx, so keep in mind to update it here as well.

Reply:
https://github.com/Dasharo/coreboot/pull/584/commits/0a86bed426097463f5239885fb216d30bd3aa597

It seems both paltforms will have the same igpu configuration, so it can be moved to shared devicetree

Reply:
[unified](https://github.com/Dasharo/coreboot/pull/576/commits/fa4a0608e35f397cf07bb76f0d22d972abe5a43e#diff-ee387f153c732f17306bdf678655b50fa130fe54c2c26bbf07ba588cee0f1259R15)